src/Infrastructure/Identity/ApplicationUser.cs

```diff
@@ -12,4 +12,7 @@ public class ApplicationUser : IdentityUser
     public DateTime RefreshTokenExpiryTime { get; set; }
 
     public string? ObjectId { get; set; }
+
+    public bool IsRootUser { get; set; }
```

Comment:
Same here... No need to put this in the database... Can just check `Where(x => EF.Property<string>(x, "TenantId") == MultitenancyConstants.Root.Id)`.

Reply:
Because if it stores it in the table the constants are used as the initialization of the system, and then you can change without being wired.

Reply:
You can not simply change the tenantId... see previous comment above...